lntest/itest/lnd_multi-hop_htlc_local_timeout_test.go

```diff
@@ -190,7 +190,7 @@ func testMultiHopHtlcLocalTimeout(net *lntest.NetworkHarness, t *harnessTest,
 
 	// We'll mine the remaining blocks in order to generate the sweep
 	// transaction of Bob's commitment output.
-	slowMineBlocks(t, net, defaultCSV, expectedTxes)
+	slowMineBlocks(t, net, defaultCSV-1, expectedTxes)
```

Comment:
Hmm, IMO all the `-1` everywhere (but not sometimes) makes the ordering in the tests at times hard to reason about. I think this could do with a comment explaining that it's a `-1` here since we mined the transaction confirming the commitment above (this is the case right?).

Reply:
It's been a while, but IIRC the main underlying issue is that the contract resolver sends the output for sweeping on the block immediately before the sweeping tx could be included in a block. So if the commitment output is eligible for inclusion in a block at `[current +] defaultCSV`, the sweeping tx is broadcast at block `[current +] defaultCSV-1` so if you want to reliably wait for that tx in the mempool, you need to do the subtraction.

When fast mining, the blocks come in faster then the contractcourt+sweeper can send the tx to the mempool, so it doesn't make a difference (modulo flakes). When slow mining, the contractcourt+sweeper have enough time to sweep any possible outputs at every block, thus you can reliably trigger the issue.